lib/IDE/Refactoring.cpp

```diff
@@ -7356,21 +7432,51 @@ class AsyncConverter : private SourceEntityWalker {
       OS << tok::kw_try << " ";
     }
     OS << "await ";
-    addRange(CE->getStartLoc(), CE->getFn()->getEndLoc(),
-             /*ToEndOfToken=*/true);
+
+    // If there's no alternative then continue, but assume that the alternative
+    // has the same name as the callee
+    auto *Alternative = HandlerDesc.Func->getAsyncAlternative();
+    ArrayRef<ParamDecl *> AlternativeParams;
```

Comment:
I think you only need this further below in a branch that has `Alternative != nullptr`, so you could initialize it there.